types/src/main/kotlin/fp/serrano/Transformative.kt

```diff
@@ -2,4 +2,4 @@ package fp.serrano
 
 @Target(AnnotationTarget.CLASS)
 @Retention(AnnotationRetention.SOURCE)
-annotation class transformative
+public annotation class transformative
```

Comment:
This is because of this: https://github.com/serras/transformative/pull/1/files#diff-9e17a79975cf38eda3f46edfc7ebb2d6d26c6352fd1a1edd988ffbf560a00e51R9